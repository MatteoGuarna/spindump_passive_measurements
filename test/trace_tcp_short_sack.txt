Short tcp connection on lossy link to trigger SACK option.