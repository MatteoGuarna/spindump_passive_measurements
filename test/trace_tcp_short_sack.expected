TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894688558 new starting packets 1 0 bytes 52 0 bandwidth 52 0
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894741073 measurement up right 52515 packets 1 1 bytes 52 52 bandwidth 52 52
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894741225 measurement up left 152 packets 1 1 bytes 52 52 bandwidth 52 52
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894776925 measurement up left 75 packets 3 2 bytes 649 104 bandwidth 649 104
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894804373 measurement up right 62529 packets 4 2 bytes 701 104 bandwidth 701 104
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894805849 measurement up left 569 packets 4 4 bytes 701 330 bandwidth 701 330
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894911135 measurement up right 105031 packets 14 4 bytes 11064 330 bandwidth 11064 330
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894931423 measurement up right 124950 packets 14 5 bytes 11064 382 bandwidth 11064 382
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894931509 measurement up right 125035 packets 14 6 bytes 11064 442 bandwidth 11064 442
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894951142 measurement up right 144494 packets 15 7 bytes 12516 502 bandwidth 12516 502
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894951221 measurement up right 144571 packets 17 8 bytes 14158 562 bandwidth 14158 562
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894998274 measurement up right 47074 packets 17 9 bytes 14158 630 bandwidth 14158 630
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470894998546 measurement up left 202 packets 17 12 bytes 14158 903 bandwidth 14158 903
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470895043757 measurement up right 45211 packets 18 14 bytes 14236 1023 bandwidth 14236 1023
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470895207196 measurement up left 40249 packets 18 16 bytes 14236 1867 bandwidth 14236 1867
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470895460839 measurement up right 49172 packets 21 17 bytes 16017 1919 bandwidth 16017 1919
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470895607344 measurement up left 40803 packets 21 19 bytes 16017 2735 bandwidth 16017 2735
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470898889386 measurement closing right 51025 packets 23 19 bytes 16097 2735 bandwidth 16057 2735
TCP 192.168.178.28 <-> 92.122.26.214 57710:443 at 1605470898889433 measurement closing left 47 packets 24 20 bytes 16137 2806 bandwidth 16057 2735
